### DISCUSSION TITLE
Any document for the current file type for notes?

### DISCUSSION THREAD

**Discussion by Iey4iej3** [LinwoodDev/Butterfly#483] [General]
I wonder the structure of the current file type for notes. It looks like a binary data file. I wonder whether it is some standard data file (such as BSON)?

**Comment by CodeDoctorDE** [comment]
Butterfly uses json files inside a zip directory

**Comment by Iey4iej3** [comment]
Then is there any performance issue when notes are large (if, say, with frequent autosave)?

**Reply by CodeDoctorDE** [reply]
I don't see a real performance problem.
The zip structure allows me to save assets inside of the document
